samples/apps/auto-anny/agent_utils.py

```diff
@@ -0,0 +1,36 @@
+from autogen import AssistantAgent, UserProxyAgent, config_list_from_json
+
+
+async def solve_task(task):
```

Comment:
Seeing this function is incredibly insightful! There are some totally non-trivial things in my opinion that it would be good to over time make sure autogen makes tasks like this much easier to implement.

Reply:
Jack do you have suggestions for functions/commands to add?

Reply:
I'm not sure at the moment - but I would really like to see how this function changes after the #1402 is merged. If the get takeaway functionality could be used?

Reply:
I would love to use that once #1402 is merged!